qualtran/testing.py

```diff
@@ -370,3 +371,18 @@ def check_bloq_example_decompose(bloq_ex: BloqExample) -> Tuple[BloqCheckResult,
         return BloqCheckResult.ERROR, f'{bloq_ex.name}: {e}'
 
     return BloqCheckResult.PASS, ''
+
+
+def assert_wire_symbols_match_expected(bloq: Bloq, expected_ws: List[str]):
+    """Assert a bloq's wire symbols match the expected ones.
+
+    Args:
+        bloq: the bloq whose wire symbols we want to check.
+        expected_ws: A list of the expected wire symbols.
+    """
+    ws = []
+    regs = bloq.signature
+    for i, r in enumerate(regs):
+        ws.append(bloq.wire_symbol(Soquet(i, r)).text)
```

Comment:
this will only work for `shape=()` situations 

And passing an integer in place of the `binst` is sortof a type error; but it's sortof an error for the wire_symbol method to actually use the `binst` field (whoops). Maybe make a note of that.

Reply:
Opened #608 to track, and a note.